docs/modules/release-notes/pages/all-releases/4_13_2S.adoc

```diff
@@ -0,0 +1,118 @@
+= Kobiton 4.13.2S release notes
+:navtitle: Kobiton 4.13.2S release notes
+
+_June 20, 2024_
+
+== New OS support
+
+Kobiton 4.13.2S adds support for the following:
+
+* iOS/iPadOS 17.4.1, 17.5 Beta, 17.5.1
+* Mac OS Sonoma 14.4.1, 14.5
+* Android 15 Beta 3
+
+[NOTE]
+====
+* At this time, we recommend hosting iOS 17+ devices via Standard mode deployments for improved performance.
+* For Android 15 Beta 3, the cleanup policy cannot remove a signed-in account.
+====
+
+== Combined HAR files for network payload data
+
+Previously, the network payload capture log was split into multiple .payload files. We can now consolidate multiple network log files into a single payload file, making it easier to convert to load test scripts on products like Grafana K6, JMeter, and Neoload for performance testing.
+
+We also introduced two options for downloading the payload data:
+
+* Include both request and response payloads.
+* Include request payloads only (reduced size and optimized for load testing)

```

Comment:
Need to remove period in first bullet or add period in second bullet to be consistent